### PR TITLE
feature/unit tests part 4

### PR DESCRIPTION
**Pull request by yilmazbahadir**
Added unit tests for the following components:
- AccountSelectorPanel
- ContactSelector
- DeleteProfileButton

### CODE REVIEW

File: __tests__/components/AccountSelectorPanel.spec.ts [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+        const stateChanges = {
+            currentAccount: account,
+            knownAccounts: [account],
+        } as Partial<AccountState>;
+
+        // Act:
+        const wrapper = getAccountSelectorPanelWrapper(stateChanges);
+
+        // Assert:
+        expect(wrapper.find('div').exists()).toBeTruthy();
```
Comment (Jaguar0625) [inline __tests__/components/AccountSelectorPanel.spec.ts:93]: why truthy vs checking against true ?

File: __tests__/components/ContactSelector.spec.ts [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        const component = wrapper.vm as ContactSelectorTs;
+
+        // Act:
+        wrapper.find('ul li').trigger('click');
+        await wrapper.vm.$nextTick();
+
+        // Assert:
+        expect(wrapper.emitted('change')[0][0]).toBe(contactList[0].id);
+        // @ts-ignore - accessing private property
+        expect(component.poptipVisible).toBe(false);
```
Comment (Jaguar0625) [inline __tests__/components/ContactSelector.spec.ts:68]: might be better to set to true in arrange, to validate action is clearing flag

File: __tests__/components/AccountSelectorPanel.spec.ts (hunk 1 above)
Comment (yilmazbahadir) [reply __tests__/components/AccountSelectorPanel.spec.ts]: updated 👍

File: __tests__/components/ContactSelector.spec.ts (hunk 2 above)
Comment (yilmazbahadir) [reply __tests__/components/ContactSelector.spec.ts:68]: updated 👍

**Review by Jaguar0625 — APPROVED** [review]

**Pull request merged**